@AH-64D Apache Longbow/Addons/fza_ah64_systems/functions/interact/fn_interactBattSwitch.sqf

```diff
@@ -18,4 +18,4 @@ Author:
 ---------------------------------------------------------------------------- */
 params ["_heli"];
 
-_heli setVariable ["fza_systems_battSwitchOn",  !(_heli getVariable "fza_systems_battSwitchOn")];
+_heli setVariable ["fza_systems_battSwitchOn",  !(_heli getVariable "fza_systems_battSwitchOn"), true];
```

Comment:
Should't this use the new function?

Reply:
the logic flip setting of the variable would make the new func mute, as its always going to be different when set